pages/api/_common.ts

```diff
@@ -1,4 +1,11 @@
-import { ethers, Contract } from "ethers";
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
We can't avoid breaking these rules here because of the limitations of the types available to us.